projects/aca-content/src/lib/components/search/search-results-row/search-results-row.component.spec.ts

```diff
@@ -91,50 +83,100 @@ describe('SearchResultsRowComponent', () => {
 
     fixture = TestBed.createComponent(SearchResultsRowComponent);
     component = fixture.componentInstance;
+    utils = new UnitTestingUtils(fixture.debugElement);
   });
 
   it('should show the current node', () => {
     component.context = { row: { node: nodeEntry } };
     fixture.detectChanges();
 
-    const element = fixture.nativeElement.querySelector('div');
-    expect(element).not.toBeNull();
+    expect(utils.getByCSS('div')).not.toBeNull();
   });
 
   it('should correctly parse highlights', (done) => {
     component.context = { row: { node: resultEntry } };
-    component.content$
-      .asObservable()
-      .pipe(first())
-      .subscribe(() => {
-        fixture.detectChanges();
-
-        const nameElement: HTMLSpanElement = fixture.debugElement.query(By.css('.aca-link.aca-crop-text')).nativeElement;
-        expect(nameElement.innerHTML).toBe('<span class="aca-highlight">Random</span>');
-        expect(nameElement.title).toBe('Random');
-
-        const titleElement: HTMLSpanElement = fixture.debugElement.query(By.css('[data-automation-id="search-results-entry-title"]')).nativeElement;
-        expect(titleElement.innerHTML).toBe(' ( <span class="aca-highlight">Random</span> title )');
-        expect(titleElement.title).toBe('Random title');
-
-        const descriptionElement: HTMLDivElement = fixture.debugElement.query(
-          By.css('[data-automation-id="search-results-entry-description"]')
-        ).nativeElement;
-        expect(descriptionElement.innerHTML).toBe('some <span class="aca-highlight">random</span> description');
-        expect(descriptionElement.title).toBe('some random description');
-
-        const contentElement: HTMLDivElement = fixture.debugElement.query(By.css('.aca-result-content.aca-crop-text')).nativeElement;
-        expect(contentElement.innerHTML).toBe('...Interesting <span class="aca-highlight">random</span> content...');
-        expect(contentElement.title).toBe('...Interesting random content...');
-        done();
-      });
+    component.content$.pipe(first()).subscribe(() => {
+      fixture.detectChanges();
+
+      expect(utils.getByCSS('.aca-link.aca-crop-text').nativeElement.innerHTML).toBe('<span class="aca-highlight">Random</span>');
```

Comment:
That selector .aca-link.aca-crop-text is repeated multiple times so can be wrapped into reusable function